.circleci/test-deploy.yml

```
@@ -112,7 +112,7 @@ workflows:
           working-directory: examples/angular-app
           cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/angular-app/package.json" }}
           start-command: "npm run start"
-          cypress-command: "npx wait-on@latest http://localhost:4200 && npx cypress run --browser chrome"
+          cypress-command: "npx wait-on@latest http://localhost:4200 && npx cypress run --record false --browser chrome"
```

We recommend removing the `--record` key over passing `false` - this is technically invalid, and there are some known bugs if this is passed.

```suggestion
          cypress-command: "npx wait-on@latest http://localhost:4200 && npx cypress run --browser chrome"
```

---

Your suggestion returns the workflow to its original state which causes an warning with the text:

```text
If you don't want to record these runs, you can silence this warning:

  $ cypress run --record false
```

It would be difficult to remove the record key. I assume this must be stored as an environment value `CYPRESS_RECORD_KEY` in the CircleCI environment variables, and it is needed to record the Angular CT tests.

Could we just record the Angular E2E test as well as the Angular CT tests? In this case the change to the command line would be to have it saying
```shell
npx cypress run --record --browser chrome
```